chain/jsonrpc/src/lib.rs

```diff
@@ -384,6 +384,9 @@ impl JsonRpcHandler {
             "next_light_client_block" => {
                 process_method_call(request, |params| self.next_light_client_block(params)).await
             }
+            "light_client_block_proof" => {
```

Comment:
```suggestion
            "EXPERIMENTAL_light_client_block_proof" => {
```

We have a kind of convention that all newly introduced RPC methods must have an `EXPERIMENTAL_` prefix.

After a few release cycles, if the method has proven to stay unchanged, we are ready to promote it to a regular one (w/o the prefix).

Reply:
Fixed in 9768ccc58795502db2d171cd5fc4f5011a3197d5